packages/grid/x-data-grid-generator/src/renderer/renderEditCurrency.tsx

```diff
@@ -17,17 +17,19 @@ const StyledAutocomplete = styled(Autocomplete)(({ theme }) => ({
       height: '100%',
     },
   },
-}));
+})) as typeof Autocomplete;
 
 function EditCurrency(props: GridRenderEditCellParams<string>) {
   const { id, value, field } = props;
 
   const apiRef = useGridApiContext();
 
-  const handleChange = React.useCallback(
+  const handleChange = React.useCallback<
+    NonNullable<AutocompleteProps<string, false, true, false>['onChange']>
+  >(
     (event, newValue) => {
       apiRef.current.setEditCellValue({ id, field, value: newValue.toUpperCase() }, event);
-      if (!event.key) {
+      if (!(event as any).key) {
```

Comment:
The `event` is not typed by the `AutocompleteProps`?

Reply:
@m4theushw can answer here
According to TS, `event.key` does not exisst
Is the typing of the core broken or is this check useless ?